src/libecalc/presentation/yaml/mappers/component_mapper.py

```diff
@@ -327,13 +266,29 @@ def from_yaml_to_dto(self, data: Dict) -> dto.Installation:
             )
             for fuel_consumer in data.get(EcalcYamlKeywords.fuel_consumers, [])
         ]
-        venting_emitters = [
-            self.__venting_emitters_mapper.from_yaml_to_dto(
-                venting_emitters,
-                regularity=regularity,
-            )
-            for venting_emitters in data.get(EcalcYamlKeywords.installation_venting_emitters, [])
-        ]
+
+        venting_emitters = []
+        for venting_emitter in data.get(EcalcYamlKeywords.installation_venting_emitters, []):
+            emission = venting_emitter.get(EcalcYamlKeywords.emission)
+            emission_rate = emission.get(EcalcYamlKeywords.rate)
+
+            try:
+                venting_emitters.append(
+                    YamlVentingEmitter(
+                        name=venting_emitter.get(EcalcYamlKeywords.name),
+                        emission=YamlVentingEmission(
+                            name=emission.get(EcalcYamlKeywords.name),
+                            rate=YamlRate(
+                                value=emission_rate.get(EcalcYamlKeywords.value),
+                                unit=emission_rate.get(EcalcYamlKeywords.unit),
+                                rate_type=emission_rate.get(EcalcYamlKeywords.rate_type),
+                            ),
+                        ),
+                        user_defined_category=venting_emitter.get(EcalcYamlKeywords.user_defined_tag),
+                    )
```

Comment:
```suggestion
                    YamlVentingEmitter(
                        **venting_emitter
                    )
```

We should be able to do this?

Reply:
I will check!

Reply:
Done. It works, but need to convert `CATEGORY` to `user_defined_category` for matching to work. This is a workaround to avoid renaming user_defined_category to category all places. This should probably be done later though.

Reply:
Changed it, now user_defined_category is implemented as a property in YamlVentingEmitter.